pyproject.toml

```diff
@@ -56,6 +57,22 @@ exclude = '''
 )/
 '''
 
+[tool.pyright]
```

Comment:
Nice :) I have not started to use  `pyright` or `ruff` yet but its a nice addition !